api/tag/schema.py

```diff
@@ -22,15 +24,24 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, **kwargs):
-        validate_empty_fields(**kwargs)
-        tag = TagModel(**kwargs)
-        payload = {
-            'model': TagModel, 'field': 'name', 'value':  kwargs['name']
-            }
-        with SaveContextManager(
-           tag, 'Tag', payload
-        ):
-            return CreateTag(tag=tag)
+        try:
+            validate_empty_fields(**kwargs)
+            tag = TagModel(**kwargs)
+            payload = {
+                'model': TagModel, 'field': 'name', 'value':  kwargs['name']
+                }
+            query = Tag.get_query(info)
+            result = query.filter(TagModel.state == "active")
+            tag_name = result.filter(
```

Comment:
kindly remove this code because its redundant with the presence of the context manager

Reply:
Same as above @RachaelNantale